.gitignore

```diff
@@ -133,9 +133,9 @@ dmypy.json
 .idea/
 .vscode/
 
-dialogues/*/data/*
+#dialogues/*/data/*
```

Comment:
We generate a lot of files in these folders when running the scripts in this repo. This is to avoid accidental commits. If you are sure you want to commit a dataset, you can do so by `git add -f {file_name}`.